lldb/test/API/lang/objc/real-definition/TestRealDefinition.py

```diff
@@ -27,13 +27,11 @@ def test_frame_var_after_stop_at_interface(self):
         # Run at stop at main
         lldbutil.check_breakpoint(self, bpno=1, expected_hit_count=1)
 
-        self.runCmd("settings set target.prefer-dynamic-value no-dynamic-values")
-
         # This should display correctly.
         self.expect(
             "frame variable foo->_bar->_hidden_ivar",
             VARIABLES_DISPLAYED_CORRECTLY,
-            substrs=["(NSString *)", "foo->_bar->_hidden_ivar = 0x"],
+            substrs=["foo->_bar->_hidden_ivar = 0x"],
```

Comment:
FWIW, this would be nicer if it were implemented using the SBAPI instead of expect. Maybe we should upstream `lldbutil.check_variable` from the Swift branch?

Reply:
yea can do that as a follow-up